packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -249,6 +279,14 @@ class ReorderableListView extends StatefulWidget {
   /// {@macro flutter.widgets.scrollable.dragStartBehavior}
   final DragStartBehavior dragStartBehavior;
 
+  /// The delay after which the user is able to drag the element.
+  /// By default, this is [kLongPressTimeout]. If set to `null`, and having
+  /// [buildDefaultDragHandlesOnMobile] as `false`, the list items are not going
+  /// to be reorderable by default.
+  ///
+  /// This only applies to mobile platforms.
+  final Duration? delayedDragStartDuration;
```

Comment:
While I think adding a delay parameter to `ReorderableDelayedDragStartListener` is a reasonable change, I am not sure it is needed at this level. It would only apply in the case where ` buildDefaultDragHandles` is true and you are on a mobile device. And again, if applications want to change this delay they could turn off the default handles and wrap the items with a `ReorderableDelayedDragStartListener` with the delay they want (given that you have exposed the param below).